js/my-github.js

```diff
@@ -1 +1,70 @@
-// Write code here to communicate with Github
+var xClick = document.getElementById("repos-count");
+var ownerName = document.getElementById("reposOwner");
+fetch("https://api.github.com/users/milleymilli/repos")
+  .then(response => {
+    return response.json();
+  })
+  .then(data => {
+    // console.log(data.length);
+    xClick.innerHTML = data.length;
+    let reposi = "";
+    data.forEach(element => {
```

Comment:
This is correct. This is an alternative approach which uses an external function.
```suggestion
            json.forEach(element => {
                 const item = createItem(element.full_name, element.html_url);
                 repoList.appendChild(item);
```

Reply:
yes i can see ,,,but won't be a link after u append it. It should be a link so we have to put element.html_url in href attribute i guess

Reply:
Yes, it is a link after appending it. This is the code for createItem.
```
function createItem(name, url) {
    const item = document.createElement("li");
    const link = document.createElement("a");
    item.appendChild(link);
    link.textContent = name;
    link.setAttribute("href", url);
    return item;
}
```